src/Avalonia.Controls/Primitives/SnapPointsAlignment.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Avalonia.Controls.Primitives
+{
+    public enum SnapPointsAlignment
```

Comment:
Can you add documentation for these new types? IScrollSnapPointsInfo, IScrollSnapPointsInfo, IScrollSnapPointsInfo.

Other than that, looks good to me.